themes/navy/source/js/main.js

```diff
@@ -1,5 +1,187 @@
 $(document).ready(function () {
 
+    var allPosts = [],
+        deferred,
+        deferreds = [];
+
+    var allBlogs = [
+        {
+            url: 'https://vac.dev/feed.xml',
+            blog: 'vac',
+            type: 'xml'
+        },
+        {
+            url: 'https://our.status.im',
+            key: '2abb4728e472593059746cc5e8',
+            version: 'v2',
+            limit: 3,
+            blog: 'our-status',
+            type: 'ghost'
+        },
+        {
+            url: 'https://news.nimbus.team',
+            key: '1785a1f63f60077dbe3156d1b5',
+            version: 'v2',
+            limit: 3,
+            blog: 'nimbus',
+            type: 'ghost'
+        },
+        // {
+        //     url: 'https://news.keycard.tech',
+        //     key: 'd7ad0ac3484bdf01fadf32b59c',
+        //     version: 'v2',
+        //     limit: 3,
+        //     blog: 'keycard',
+        //     type: 'ghost'
+        // },
+        {
+            url: 'https://news.thestatus.network',
+            key: '66d8d4f364b9f46c90c9d45bf4',
+            version: 'v2',
+            limit: 3,
+            blog: 'status-network',
+            type: 'ghost'
+        },
+        {
+            url: 'https://embark.status.im/atom.xml',
+            blog: 'embark',
+            type: 'xml'
+        }
+    ];
+
+    function loadGhostPosts(arr){
+
+        var ghostAPI = new GhostContentAPI({
+            url: arr.url,
+            key: arr.key,
+            version: arr.version
+        });
+
+        return ghostAPI.posts
+            .browse({
+                include: 'authors',
+                fields: ['title', 'url', 'published_at'],
+                limit: arr.limit
+            })
+            .then(function(posts){
+
+                $.each(posts, function (i,v) { 
```

Comment:
Why are you abusing jQuerys `each()` so much? As far as I know the support of `for (const x of y)` is good:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
Only IE doesn't support it.

Reply:
Replaced jQuery's `each` with `for`